source/archive/doc/design/exe/E04.html

```diff
@@ -713,10 +713,10 @@ <h3>This file is part of the Perl 6 Archive</h3>
 <p>except that its parameter list doesn't require parentheses. That
 implies:</p>
 <ul>
-<li><a name='The Perl 6 for, while, if, and given statements each take two arguments: an expression that controls them and a subroutine/closure that they execute. Normally, that closure is just a block (in Perl6 all blocks are really closures):'></a>The Perl 6 <code>for</code>, <code>while</code>, <code>if</code>, and <code>given</code> statements each take
+<li><a name='The Perl 6 for, while, if, and given statements each take two arguments: an expression that controls them and a subroutine/closure that they execute. Normally, that closure is just a block (in Raku all blocks are really closures):'></a>The Perl 6 <code>for</code>, <code>while</code>, <code>if</code>, and <code>given</code> statements each take
 two arguments: an expression that controls them and a
 subroutine/closure that they execute. Normally, that closure is just a
-block (in Perl6 <i>all</i> blocks are really closures):</li>
+block (in Raku <i>all</i> blocks are really closures):</li>
```

Comment:
Most probably not here, even less so if you don't do it consistently, changing only Perl6 but not Perl 6